src/settings/TargetProfileDetail.test.js

```diff
@@ -1,47 +1,144 @@
 import React from 'react';
-import { screen } from '@testing-library/react';
-import { MemoryRouter } from 'react-router-dom';
+import { BrowserRouter } from 'react-router-dom';
+
+import { buildResources } from '@folio/stripes-data-transfer-components/test/helpers';
```

Comment:
@mariia-aloshyna are we bringing the entire `@folio/stripes-data-transfer-components` as a dependecy just to be able to use `buildResources` in tests here? This seems very heavy.

Reply:
Hi @mkuklis Agree. Created [PR](https://github.com/folio-org/ui-inventory/pull/1936) to get rid of this dep